src/snowflake/connector/vendored/urllib3/response.py

```diff
@@ -17,6 +17,23 @@
 except ImportError:
     brotli = None
 
+try:
+    import zstandard as zstd
```

Comment:
I'm interested in under what circumstances will the backend send back zstd compressed data, is there a configuration?

also what error would connector trigger if it receives zstd compressed data but zstd is not installed in the execution env